buildSrc/src/main/groovy/com/uber/okbuck/composer/android/AndroidTestRuleComposer.groovy

```diff
@@ -55,7 +57,7 @@ final class AndroidTestRuleComposer extends AndroidBuckRuleComposer {
                 .options(target.main.jvmArgs)
                 .jvmArgs(target.testOptions.jvmArgs)
                 .env(target.testOptions.env)
-                .robolectricManifest(fileRule(target.manifest))
+                .robolectricManifest(manifestRuleName != null ? manifestRuleName : fileRule(target.manifest))
```

Comment:
lets just use `manifest(target)` here directly

We should remove `manifestRuleName`